spiketools/plts/spatial.py

```diff
@@ -84,18 +84,18 @@ def plot_positions(positions, spike_positions=None, landmarks=None,
 
 @savefig
 @set_plt_kwargs
-def plot_position_by_time(times, positions, spike_times=None, spike_positions=None,
+def plot_position_by_time(timestamps, position, spikes=None, spike_positions=None,
                           ax=None, **plt_kwargs):
     """Plot the position across time for a single dimension.
 
     Parameters
     ----------
-    times : 1d array
-        Time values associated with the position values.
-    positions : 1d array
+    timestamps : 1d array
+        Timestamps, in seconds, corresponding to the position values.
+    position : 1d array
         Position values, for a single dimension.
-    spike_times : 1d array, optional
-        Timepoints at which spikes occur.
+    spikes : 1d array, optional
```

Comment:
In `sim_spiketimes` we use `spike_times` to refer to the return variable. Should we change here (`spikes` -> `spike_times`) since the two have the same meaning?

Reply:
Right now, we default to `spikes` as a label across the module, which expects spike times, but within the `sim` sub-module use `spike_times` and `spike_train` more explicitly. I guess the more broadly consistent would be to use `spike_times` and `spike_train` consistently everywhere.